### PR TITLE
Changelog v3.0.0

### PR DESCRIPTION
**Pull request by j0tunn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,31 @@
...
 
+## 3.0.0 - 2016-03-23
+
+* Fix `CLIENT_STOPPED_SESSION` bug during performing setWindowSize in post actions (@j0tunn)
+* Suite.browsers: ability to set browsers, in which suite should be run (@egavr)
+* Added ability to use .js/.json config files (@j0tunn)
+* Android support:
+  * Do zoom reset after page load instead of before screenshot (@SwinX)
+  * Fixed calibration for android emulators (@SwinX)
+  * Fixed image crop for big pixelRation (@SwinX)
```
Comment (sipayRT) [inline CHANGELOG.md]: pixelRatio

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,31 @@
 # Changelog
 
+## 3.0.0 - 2016-03-23
+
+* Fix `CLIENT_STOPPED_SESSION` bug during performing setWindowSize in post actions (@j0tunn)
```
Comment (SwinX) [inline CHANGELOG.md]: ```Fix `CLIENT_STOPPED_SESSION` bug during performing setWindowSize in post actions (@j0tunn)``` -> ```Fixed `CLIENT_STOPPED_SESSION` bug when performing setWindowSize in post actions (@j0tunn)```

**Comment by SwinX** [comment]
:ok:

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,31 @@
...
+  * Fixed image crop for big pixelRation (@SwinX)
+* Enable uncaughtException logging (@SwinX)
+* Fix coverage (@benedfit):
+  * pass in all original options when processing media rules
+  * check length of selector after pseudo-elements have been removed
+
+**BREAKING CHANGES**:
+
+* Removed skip method for actions (@j0tunn)
+* Accumulate before and after actions for nested suites (@j0tunn)
```
Comment (sipayRT) [inline CHANGELOG.md]: Accumulate "before" and "after" actions for nested suites

**Comment by sipayRT** [comment]
ok

**Pull request merged**